CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ You can also check the [release page](https://github.com/visualize-admin/visuali
 
 ## Unreleased
 
+- Performance
+  - Dimension values are now fetched in parallel in case of fetching for several dimensions at once
+  - Refactored shapes and coordinates fetching to not rely on resolver chains
```

Comment:
Was it not performant because we would first load the dimension, and then the shapes ? And now we do not have to wait for the dimension ?

Reply:
Technically the improvement is not really noticeable, but now we have one resolver for cube, dimension and shapes, instead of the situation visible in the screenshot. We used to have lots of resolver chains, but at some point removed them and switched to scalars to improve things a bit, this was the last remaining piece of code to adapt the same pattern :)

<img width="280" alt="Screenshot 2024-03-15 at 13 55 41" src="https://github.com/visualize-admin/visualization-tool/assets/52032047/0d03ca7f-94aa-456d-b799-23176fcc32d1">

Here it's a first step towards optimizing it further and again, not fetch the whole shape, but rather "picked" properties to really gain visible improvement. But this was more complex to refactor, as we need the whole shape in several methods that are used in this query, so could be something to focus on in the next sprint 👍